lib/tapyrus/script/script.rb

```diff
@@ -130,6 +130,12 @@ def self.parse_from_addr(addr)
           Tapyrus::Script.to_p2pkh(hex)
         when Tapyrus.chain_params.p2sh_version
           Tapyrus::Script.to_p2sh(hex)
+        when Tapyrus.chain_params.cp2pkh_version
+          color = Tapyrus::Color::ColorIdentifier.nft(Tapyrus::OutPoint.new("01" * 32, 1))
+          Tapyrus::Script.to_cp2pkh(color, hex)
+        when Tapyrus.chain_params.cp2sh_version
+          color = Tapyrus::Color::ColorIdentifier.nft(Tapyrus::OutPoint.new("01" * 32, 1))
+          Tapyrus::Script.to_cp2sh(color, hex)
         else
           throw e
         end
```

Comment:
With the addition of this implementation, you need to test that the color ID is decoded as the intended data.